libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -686,6 +690,24 @@ void PbftManager::loopBackFinishState_() {
   current_step_clock_initial_datetime_ = std::chrono::system_clock::now();
 }
 
+void PbftManager::broadcastVotes(bool rebroadcast) {
+  auto [round, period] = getPbftRoundAndPeriod();
+  auto net = network_.lock();
+  if (net) {
+    if (auto soft_voted_block_data = getTwoTPlusOneSoftVotedBlockData(period, round);
```

Comment:
I think we need to call it because there might be votes coming in the meantime. This will not be called that often. Maybe once each 30 seconds so I think it should be fine